index.html

```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
                                     </ul>
                                 </div>
                             </div>
+                        </article> -->
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="https://pbs.twimg.com/profile_images/1755583699/me-smaller_400x400.JPG" style="width:100px;height:100px;"/>
+                                </div>
+                                <div class="media__body">
+                                    <h4 class="media__body__heading">Anthony Menasse</h4>
+                                    <ul class="media__body__links">
+                                        <li><a href="https://twitter.com/sysantmin">@sysantmin</a></li>
+                                        <li><a href="#">github/sysantmin</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                      </article>
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="https://pbs.twimg.com/profile_images/2958764660/2f6cda1ee4c8b55ce384b2e9f3fe5ac6_400x400.png" style="width:100px;height:100px;"/>
+                                </div>
+                                <div class="media__body">
+                                    <h4 class="media__body__heading">Colin Rolfe</h4>
+                                    <ul class="media__body__links">
+                                        <li><a href="https://twitter.com/colin_rolfe">@colin_rolfe</a></li>
+                                        <li><a href="#">github/colin_rolfe</a></li>
+                                    </ul>
+                                </div>
+                            </div>
+                      </article>
+
+                      <article class="past-speaker Grid-cell">
+                            <div class="media">
+                                <div class="media__figure">
+                                    <img src="http://musings.tinbrain.net/static/img/banner.png" style="width:100px;height:100px;"/>
```

Comment:
A better profile image for @funkybob is his Github one:
https://avatars1.githubusercontent.com/u/301363?v=3&s=80

Notice that you can do `s=80` on the end to get a version that's 80x80 pixels.